frontend/src/helper/authError.ts

```diff
@@ -0,0 +1,15 @@
+import { AxiosError } from "axios";
+import { AuthTokenResponse, ErrorResponse, AuthFlow } from "../types/AuthTypes";
+
+export const getAuthErrMessage = (axiosErrRes: AxiosError["response"], flow: AuthFlow): string => {
+  if (axiosErrRes && axiosErrRes.data && axiosErrRes.data.error) {
+    return axiosErrRes.data.error;
+  }
+  return `Error ${flow === 'LOGIN' ? "logging in" : "signing up"}. Please try again later.`;
+}
```

Comment:
Takes in the auth flow (either 'LOGIN' or 'SIGNUP') from which we're accessing this method, as to consolidate the two functiosn we had before into one